### ISSUE TITLE
Vimtex: Xdotool not available on MacOs

### ISSUE THREAD

**Issue by b-m-f** [nix-community/nixvim#634]
https://github.com/nix-community/nixvim/blob/934bf7e2e3770ad7302e3ac90c2011dde72ca8e7/plugins/languages/vimtex.nix#L41
```
       error: Package ‘xdotool-3.20211022.1’ in /nix/store/kswx44k2ii4svvr27qlyn41jkyik889l-source/pkgs/tools/X11/xdotool/default.nix:25 is not available on the requested hostPlatform:
         hostPlatform.config = "aarch64-apple-darwin"
         package.meta.platforms = [
           "aarch64-linux"
           "armv5tel-linux"
           "armv6l-linux"
           "armv7a-linux"
           "armv7l-linux"
           "i686-linux"
           "loongarch64-linux"
           "m68k-linux"
           "microblaze-linux"
           "microblazeel-linux"
           "mips-linux"
           "mips64-linux"
           "mips64el-linux"
           "mipsel-linux"
           "powerpc64-linux"
           "powerpc64le-linux"
           "riscv32-linux"
           "riscv64-linux"
           "s390-linux"
           "s390x-linux"
           "x86_64-linux"
         ]

```

How about adding a new option to choose the [vimtex view method](https://github.com/lervag/vimtex/blob/03c83443108a6984bf90100f6d00ec270b84a339/doc/vimtex.txt#L5926).

- `zathura` could be the default and install `xdotool`
 - `zathura_simple` would not need `xdotool`

I can add an MR if this solution sounds good :)

**Comment by traxys** [comment]
I think exposing a way to change the view method is the way to go!

**Comment by b-m-f** [comment]
Thanks for the feedback. 
I'll probably find some time for the MR tmrw.

**Comment by b-m-f** [comment]
Added an MR.

It is a bit more than what we have discussed here. 
I could split that out if texlive installation is not desired. I just thought that it would be nice to have everything running out of the box.

**Comment by GaetanLepage** [comment]
I guess that this can be closed following the merge of #637 .
